cmd/buildah/from.go

```diff
@@ -67,6 +68,97 @@ var (
 	}
 )
 
+func onBuild(builder *buildah.Builder) error {
+	ctr := 0
+	for _, onBuildSpec := range builder.OnBuild() {
+		ctr = ctr + 1
+		commands := strings.Split(onBuildSpec, " ")
+		command := commands[0]
+		args := commands[1:]
+		fmt.Fprintf(os.Stderr, "STEP %d: %s\n", ctr, onBuildSpec)
+		switch command {
+		case "ADD":
+		case "COPY":
+			dest := ""
+			size := len(args)
+			if size > 1 {
+				dest = args[size-1]
+				args = args[:size-1]
+			}
+			if err := builder.Add(dest, false, buildah.AddAndCopyOptions{}, args...); err != nil {
+				return err
+			}
+			continue
+		case "ANNOTATION":
+			annotation := strings.SplitN(args[0], "=", 2)
+			if len(annotation) > 1 {
+				builder.SetAnnotation(annotation[0], annotation[1])
+			} else {
+				builder.UnsetAnnotation(annotation[0])
+			}
+			continue
+		case "CMD":
+			builder.SetCmd(args)
+			continue
+		case "ENV":
+			env := strings.SplitN(args[0], "=", 2)
+			if len(env) > 1 {
+				builder.SetEnv(env[0], env[1])
+			} else {
+				builder.UnsetEnv(env[0])
+			}
+			continue
+		case "ENTRYPOINT":
+			builder.SetEntrypoint(args)
+			continue
+		case "EXPOSE":
+			builder.SetPort(strings.Join(args, " "))
+			continue
+		case "HOSTNAME":
+			builder.SetHostname(strings.Join(args, " "))
+			continue
+		case "LABEL":
+			label := strings.SplitN(args[0], "=", 2)
+			if len(label) > 1 {
+				builder.SetLabel(label[0], label[1])
+			} else {
+				builder.UnsetLabel(label[0])
+			}
+			continue
+		case "MAINTAINER":
+			builder.SetMaintainer(strings.Join(args, " "))
+			continue
+		case "ONBUILD":
+			builder.SetOnBuild(strings.Join(args, " "))
+			continue
+		case "RUN":
+			if err := builder.Run(args, buildah.RunOptions{}); err != nil {
+				return err
+			}
+			continue
+		case "SHELL":
+			builder.SetShell(args)
+			continue
+		case "STOPSIGNAL":
+			builder.SetStopSignal(strings.Join(args, " "))
+			continue
+		case "USER":
+			builder.SetUser(strings.Join(args, " "))
+			continue
+		case "VOLUME":
+			builder.AddVolume(strings.Join(args, " "))
+			continue
+		case "WORKINGDIR":
+			builder.SetWorkDir(strings.Join(args, " "))
+			continue
+		default:
+			logrus.Errorf("OnBuild error %q invalid", onBuildSpec)
```

Comment:
This error seems really poor - can you reword it to something like `OnBuild command %q is invalid`?